### PR TITLE
releng: Image promotion for v1.19.2/v1.19.3-rc.0 - v1.17.12/v1.17.13-rc.0 - v1.18.9/v1.18.10-rc.0

### PR DESCRIPTION
**Pull request by cpanato**
### 1.19
Logs from the cloudbuild:
- [Stage](https://console.cloud.google.com/cloud-build/builds/0c6f42c6-69fd-4350-88f5-519a1479daf7?project=kubernetes-release-test)

Command to generate the config file used:
 ```shell
cip-mm --base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.19.2

cip-mm --base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.19.3-rc.0
```

### 1.17
Logs from the cloudbuild:
- [Stage](https://console.cloud.google.com/cloud-build/builds/a7c01498-4c04-4888-8f01-0543e457592c?project=kubernetes-release-test)

Command to generate the config file used:
 ```shell
cip-mm --base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.17.12

cip-mm --base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.17.13-rc.0
```

### 1.18
Logs from the cloudbuild:
- [Stage](https://console.cloud.google.com/cloud-build/builds/91f60b08-c647-4cc2-ada9-ffc4f51cb737?project=kubernetes-release-test)

Command to generate the config file used:
 ```shell
cip-mm --base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.18.9

cip-mm --base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.18.10-rc.0
```

removed the part of the mock images.

@kubernetes/release-engineering
/assign @justaugustus @saschagrunert @hasheddan  @xmudrii

### CODE REVIEW

**Comment by cpanato** [comment]
Will add the other Patch releases as well in this PR

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml [hunk 1]
```diff
@@ -5,8 +5,10 @@
     "sha256:10e2f4cc76f8f370f754ca2fe6be82eed819a31a5e639820463c671458e4014f": ["v1.16.14"]
     "sha256:11ce05565dac3f7a6562b424828b86ae194a3622592a59574b7bf603496ad1e2": ["v1.18.8"]
     "sha256:288138b199735aa767b042a2bbdc211fb852bf2e5e8368ab45e42fa1ddb0373f": ["v1.19.0-rc.3"]
+    "sha256:2c1be30e12fd4b99b87d36c758e92d6446a43c9d837a8b84c1dce36fd74579e9": ["v1.19.2"]
```
Comment (saschagrunert) [inline k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:8]: @puerco did we change something to the tooling to sort the map (…an ordered map…)? I would love to see the latest tags on top of the list.

**Review by saschagrunert — APPROVED** [review]
/lgtm
/approve

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml (hunk 1 above)
Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:8]: i did a PR some time ago and Linus made a comment: https://github.com/kubernetes-sigs/k8s-container-image-promoter/pull/251

**Comment by justaugustus** [comment]
/approve cancel
(this shouldn't go in until it includes the 1.18 images as well.)

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml (hunk 1 above)
Comment (justaugustus) [reply k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:8]: So no one has to jump to another PR...
Manifests created/updated by [`cip-mm`](https://github.com/kubernetes-sigs/k8s-container-image-promoter/tree/master/cmd/cip-mm) are sorted by digest, not tag, as the digest is a unique ID and multiple tags could be assigned to it.

This is working as intended, though the human view is maybe not ideal.
Linus opened an issue for discussion here: https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/254

**Review by hasheddan — APPROVED** [review]
/lgtm

**Review by xmudrii — APPROVED** [review]
/lgtm

**Comment by justaugustus** [comment]
/lgtm
/approve

Lift the hold once we've confirmed Google Build Admin for the day.

**Comment by justaugustus** [comment]
/hold cancel
Ben is on deck to build debs/rpms.

**Pull request merged**